include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -231,7 +234,10 @@ struct transform_reduce
         {
             const _Size __items_to_process = __adjusted_n - __adjusted_global_id;
             // Keep these statements in the same scope to allow for better memory alignment
-            new (&__res.__v) _Tp(std::move(__unary_op(__adjusted_global_id, __acc...)));
```

Comment:
I believe that `std::move` call isn't required in this and the same places in this file because the result of `__unary_op` call already is [pr-value](https://en.cppreference.com/w/cpp/language/value_category).

Reply:
I do. Thanks for catching this.

Reply:
I have fixed this point here.